Data/PalaverDbContext.cs

```diff
@@ -222,7 +222,7 @@ public async Task MarkCommentReadByUser(int threadId, int commentId, int userId)
     }
 
     public async Task<List<Comment>> Search(string searchText) {
-        return await Comments.FromSqlRaw("SELECT * FROM search_comments({0})", new NpgsqlParameter("@searchText", searchText))
+        return await Comments.FromSqlRaw("SELECT * FROM search_comments({0}) LIMIT 5", new NpgsqlParameter("@searchText", searchText))
```

Comment:
The search was already configured to only display 5 results and we have enough data now that this was returning 1100+ comments if you search for "test".  This can be changed to be more flexible if/when pagination is added to the results.